src/actions.ml

```diff
@@ -257,6 +257,55 @@ let send_doc_url ~bot_info ~github_repo_full_name job_info =
   | _ ->
       Lwt.return_unit
 
+let send_bench_url ~bot_info job_info (gh_owner, gh_repo) ~external_id =
+  Lwt_io.print "Pushing status check for bench job.\n"
+  <&>
+  let url = f "https://gitlab.com/coq/coq/-/jobs/%d" job_info.build_id in
+  let state = job_info.build_status in
+  let context = "bench" in
+  url |> Uri.of_string |> Client.get
+  >>= fun (resp, _) ->
+  let status_code = resp |> Response.status |> Code.code_of_status in
+  if Int.equal 200 status_code then
```

Comment:
FTR, I get it that you've copied this code from `send_doc_url`, but it is completely irrelevant for this function: in the case of `send_doc_url`, we check that the artifact was correctly deployed. Here, we do not need to do this because there is no artifact that we point to. Even the name of the function `send_bench_url` is a bit misleading in this regard.

Reply:
Ok i've changed the name to update_bench_status and removed all this stuff.